scripts/check_unit.sh

```diff
@@ -8,10 +8,28 @@ set -e
 
 echo "Running $0"
 
-# Packages to exclude
+go generate ./...
+pwd=`pwd`
+echo "" > coverage.txt
+
+# Running aries-framework-go unit test
 PKGS=`go list github.com/hyperledger/aries-framework-go/... 2> /dev/null | \
                                                  grep -v /mocks`
+go test $PKGS -count=1 -race -coverprofile=profile.out -covermode=atomic -timeout=10m
```

Comment:
Suggest making a bash function to get rid of the copy & paste.

Reply:
done